src/main/java/org/spongepowered/common/mixin/core/block/MixinSoundType.java

```diff
@@ -26,43 +26,60 @@
 
 import org.spongepowered.api.block.BlockSoundGroup;
 import org.spongepowered.api.effect.sound.SoundType;
+import org.spongepowered.asm.mixin.Implements;
+import org.spongepowered.asm.mixin.Interface;
+import org.spongepowered.asm.mixin.Intrinsic;
 import org.spongepowered.asm.mixin.Mixin;
 import org.spongepowered.asm.mixin.Shadow;
 
 import net.minecraft.util.SoundEvent;
 
 @Mixin(net.minecraft.block.SoundType.class)
-public abstract class MixinSoundType implements BlockSoundGroup {
+@Implements(@Interface(iface = BlockSoundGroup.class, prefix = "group$"))
+public abstract class MixinSoundType {
     @Shadow private SoundEvent breakSound;
-    @Shadow private SoundEvent stepSound;
-    @Shadow private SoundEvent placeSound;
     @Shadow private SoundEvent hitSound;
-    @Shadow private SoundEvent fallSound;
     @Shadow public abstract float getVolume();
     @Shadow public abstract float getPitch();
+    @Shadow public abstract SoundEvent getStepSound();
+    @Shadow public abstract SoundEvent getPlaceSound();
+    @Shadow public abstract SoundEvent getFallSound();
 
-    @Override
-    public SoundType getBreakSound() {
-        return (SoundType) this.breakSound;
+    @Intrinsic
```

Comment:
Forge just [removed the `SideOnly` annotation from them](https://github.com/MinecraftForge/MinecraftForge/blob/1.10.x/patches/minecraft/net/minecraft/block/SoundType.java.patch).